tests/appsec/test_traces.py

```diff
@@ -33,42 +39,48 @@ def validate_appsec_span(span):
             if "_sampling_priority_v1" not in span["metrics"]:
                 raise Exception("Metric _sampling_priority_v1 should be set on traces that are manually kept")
 
-            if span["metrics"]["_sampling_priority_v1"] != APPSEC_KEEP:
-                raise Exception(f"Trace id {span['trace_id']} , sampling priority should be {APPSEC_KEEP}")
+            MANUAL_KEEP = 2
+            if span["metrics"]["_sampling_priority_v1"] != MANUAL_KEEP:
+                raise Exception(f"Trace id {span['trace_id']} , sampling priority should be {MANUAL_KEEP}")
 
-            return True
+            return validate_appsec_span_tags(span)
```

Comment:
Useles no? `test_custom_span_tags` test it for all span.

Reply:
Those span tags are not checked otherwise. I would check them when an attack happens too.